pasqal_cloud/client.py

```diff
@@ -176,37 +177,28 @@ def _cancel_batch(self, batch_id: str) -> Dict[str, Any]:
         )["data"]
         return response
 
-    def rebatch(
-        self,
-        batch_id: str,
-        job_ids: Union[List[str], EmptyFilter],
-        status: Union[List[str], EmptyFilter],
-        min_runs: Union[int, EmptyFilter],
-        max_runs: Union[int, EmptyFilter],
-        start_date: Union[datetime, EmptyFilter],
-        end_date: Union[datetime, EmptyFilter],
+    def _rebatch(
```

Comment:
This isn't a private method if it's used outside of this file.
Many of them aren't.

Reply:
Indeed, however this is a whole different topic from the main subject of the PR but yes we should probably handle it differently. If you are fine with it I prefer to edit this in another PR more focused on the refactoring of the "strange parts" of this sdk 😄